### PR TITLE
Update drupal-org extension

### PR DESCRIPTION
**Pull request by BossElijah**
## Description

- Added command to search for Drupal projects which include themes, modules, and distributions.
- Also did minor Typescript refactor as agreed upon with the creator of the extension.

## Screencast

drupal-org-4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by BossElijah** [comment]
I don't know what is wrong with the GIT history, but for some reason it includes last PR's commit messages in this PR. What was actually changed in this PR was a bit of typescript refactor and added command to search projects.

File: extensions/drupal-org/package.json [hunk 1]
```diff
@@ -27,6 +27,13 @@
       "subtitle": "Drupal",
       "description": "Search Drupal API documentation",
       "mode": "view"
+    },
+    {
+      "name": "search-projects",
+      "title": "Search Projects",
+      "subtitle": "Drupal",
+      "description": "Search for Drupal projects",
```
Comment (hussainweb) [inline extensions/drupal-org/package.json:35]: Minor: I'd love it that someone can type in "module" or "theme" and be able to select this command. With that intent, what do you think of this change?

```suggestion
      "description": "Search for Drupal projects (modules, themes, etc)",
```

**Review by hussainweb — COMMENTED** [review]
I would have preferred a separate change for the refactor but that may not be a great idea here. I like how you have generalized the command state but wonder how to extend it if future commands need more data. That's not a showstopper here though.

Generally, I am wondering if the practice of having these multiple `util.ts` and `types.ts` is typical in TypeScript projects. I don't have a lot of experience with TypeScript. I am only raising a question out of curiosity and am open to either direction.

File: extensions/drupal-org/package.json (hunk 1 above)
Comment (BossElijah) [reply extensions/drupal-org/package.json:35]: Yeah that sounds like a great idea!

**Comment by BossElijah** [comment]
> I would have preferred a separate change for the refactor but that may not be a great idea here. I like how you have generalized the command state but wonder how to extend it if future commands need more data. That's not a showstopper here though.

I see what you mean there. I decided to do it this way in order to get it done quicker, however it would probably be best-practice to have something separate. I'd suggest we just keep it as-is for now, but note it for next time. Do you agree on that @hussainweb?

> Generally, I am wondering if the practice of having these multiple `util.ts` and `types.ts` is typical in TypeScript projects. I don't have a lot of experience with TypeScript. I am only raising a question out of curiosity and am open to either direction.

I mean, it's not necessarily a TypeScript related practice, but from my experience, it has always been easier to maintain projects that implemented it this way, especially with the `util.ts` files. If some day we want to expand the capacity of one of the commands, it will be easier to review and contribute. That being said, the `types.ts` files don't necessarily have to be in different files, I just think it's a matter of personal preference.

**Comment by hussainweb** [comment]
Yes, I understand and agree. I see you have merged my other suggestion and this LGTM.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**